### PR TITLE
Endpoint Documentation

### LINKED ISSUE #41: First draft of Milestone 4

### LINKED ISSUE #66: Milestone 4: Endpoints

### PR DESCRIPTION
**Pull request by klinkoberstar**
This PR sets up the initial doc folder structure. PR will close #41 

This PR also completes  the endpoint documentation and PR will close #66

### CODE REVIEW

**Review by MadeleineKRoberts — DISMISSED** [review]
looks good! Thank you for making this

**Comment by klinkoberstar** [comment]
@MadeleineKRoberts and @aaronhaefner: re-requesting review bc I had to run the linter on the most recent changes. I can't approve since I'm the last commit!

**Review by aaronhaefner — APPROVED** [review]
these look great and render nicely!

File: docs/decisions/architecture.md
Comment (MadeleineKRoberts) [inline docs/decisions/architecture.md:1]: I will do this.

File: docs/decisions/authentication.md
Comment (MadeleineKRoberts) [inline docs/decisions/authentication.md:1]: I will do this.

File: docs/models/model.md
Comment (MadeleineKRoberts) [inline docs/models/model.md:1]: I will do this.

File: docs/decisions/server.md
Comment (MadeleineKRoberts) [inline docs/decisions/server.md:1]: I will do this.

Comment (klinkoberstar) [reply docs/decisions/server.md:1]: This is covered by Issue #63 !

File: docs/models/model.md
Comment (klinkoberstar) [reply docs/models/model.md:1]: This is covered by issue #65 !

File: docs/decisions/authentication.md
Comment (klinkoberstar) [reply docs/decisions/authentication.md:1]: This is covered by Issue #63 !

File: docs/decisions/architecture.md
Comment (klinkoberstar) [reply docs/decisions/architecture.md:1]: This is covered by Issue #63 !

**Review by MadeleineKRoberts — APPROVED** [review]
Left some comments in the endpoints about how they are currently configured! Thank you!

**Review by MadeleineKRoberts — APPROVED** [review]
Looks good

**Pull request merged**